src/Microsoft.AspNetCore.Identity.Specification.Tests/TestRole.cs

```diff
@@ -35,6 +35,9 @@ public TestRole(string roleName) : this()
     /// <typeparam name="TKey"></typeparam>
     public class TestRole<TKey> where TKey : IEquatable<TKey>
     {
+        /// <summary>
```

Comment:
The entity types are optional, but they were shared between the two in memory implementations, I can move them back into our shared folder if you like.  The EF store tests don't use them today.